bottery/patterns.py

```diff
@@ -4,21 +4,24 @@ def __init__(self, pattern=None, *args, **kwargs):
         self.kwargs = kwargs
 
     def check(self, message):
-        message = self.clean(message)
-        return self.match(message)
+        self.message = message
+        self.full_clean()
+        return self.match(self.message)
 
-    def clean(self, message):
-        return message
+    def full_clean(self):
+        for method_name in dir(self):
+            if method_name.startswith('clean_'):
+                method = getattr(self, method_name)
+                method()
 
     def match(self, message):
         raise Exception('Method Not Implemented')
 
 
 class CaseSensitiveMixinHandler:
```

Comment:
What do you think of removing `Handler` from the mixin name?

Reply:
Sounds good to me. I also thought about dividing all Handlers in a handlers.py file and the leave only the PatternsHandler on patterns.py. I think that this will make things more organized on the future. What are your thoughts about this?